test/integration/BpmnVisualization.test.ts

```diff
@@ -79,6 +80,31 @@ describe('BpmnVisualization API', () => {
       expect(newVersion.dependencies).not.toBe(initialVersion.dependencies);
     });
   });
+
+  // The API should not fail
+  describe('Registry access when no loaded diagram', () => {
+    const bv = initializeBpmnVisualizationWithContainerId('bpmn-no-loaded-diagram');
+    const bpmnElementsRegistry = bv.bpmnElementsRegistry;
+
+    it('getElementsByIds', () => {
+      expect(bpmnElementsRegistry.getElementsByIds('fake_id')).toHaveLength(0);
+    });
+    it('getElementsByKinds', () => {
+      expect(bpmnElementsRegistry.getElementsByKinds(ShapeBpmnElementKind.CALL_ACTIVITY)).toHaveLength(0);
+    });
+    it('CSS classes', () => {
+      bpmnElementsRegistry.addCssClasses('fake_id_1', 'class1');
+      bpmnElementsRegistry.removeCssClasses('fake_id_2', 'class2');
+      bpmnElementsRegistry.toggleCssClasses('fake_id_3', 'class3');
+    });
+    it('Overlays', () => {
+      bpmnElementsRegistry.addOverlays('fake_id_1', { label: 'overlay', position: 'top-center' });
+      bpmnElementsRegistry.removeAllOverlays('fake_id_2');
+    });
+    it('updateStyle', () => {
+      bpmnElementsRegistry.updateStyle('fake_id', { fill: { color: 'red' } });
```

Comment:
**suggestion (if-minor):** I think we should add on the TS doc of these functions that they doesn't nothing if the element doesn't exist.

Reply:
Yes, good catch. I am going to check all methods that update elements, and I will update their documentation with this remark.

Reply:
done in b199bb3